src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/commands.py

```diff
@@ -67,7 +66,7 @@ def load_command_table(self, _):
         g.custom_command('remove-connector', 'k8s_uninstall_connector')
         g.custom_command('remove-dev-spaces', 'aks_remove_dev_spaces')
         g.custom_command('scale', 'aks_scale', supports_no_wait=True)
-        g.custom_command('show', 'aks_show', table_transformer=aks_show_table_format)
+        g.generic_show_command('show', 'aks_show', table_transformer=aks_show_table_format, custom_command=True)
```

Comment:
This is the role of the `getter_type` kwarg. I don't agree with providing multiple ways to do the same thing. This convenience flag only works when a module has a single custom command type. It breaks down (whereas `getter_type` does not) when custom commands are broken into multiple files (see storage and monitor).

Reply:
True yes, `getter_type` can still be used as the default to provide more flexibility. custom_command is a convenience. 
This is the same reason why we have the `command` and `custom_command` methods, you can still register a custom command using `command` and specifying a command_type.